.github/chainguard/project-board-sync.sts.yaml

```diff
@@ -0,0 +1,20 @@
+issuer: https://token.actions.githubusercontent.com
+subject: repo:chainguard-dev/project-backup:ref:refs/heads/main
+claim_pattern:
+  # Restrict to specific workflow
+  workflow_ref: chainguard-dev/project-backup/.github/workflows/sync-pr-board\.yml@refs/heads/main
+
+permissions:
+  metadata: read
+  contents: read
+  issues: read
+  pull_requests: read
+  repository_projects: read
+  organization_projects: read
```

Comment:
it adds issues/prs to a project, so we might need a write either in repository_projects or organization_projects I guess